orchagent/portsorch.cpp

```diff
@@ -7986,6 +8007,18 @@ void PortsOrch::updatePortOperStatus(Port &port, sai_port_oper_status_t status)
                     isUp ? "up" : "down");
         }
     }
+
+    if (!isUp && isSingleMemberLagPort(port))
```

Comment:
Do we have minLinks attribute? If so, we can also trigger the below change when port-channel's minLink is less than the active member count.